### ISSUE TITLE
Infoview does not load (ARM64)

### ISSUE THREAD

**Issue by PDeets** [leanprover/vscode-lean4#287]
Hello, I just installed the VS code lean4 extension, but the "Lean Infoview" is blank. I am using an ARM64 build of Windows 11, but I have the x64 build of Lean installed and am able to build and run lean programs in the VS Code terminal. The "Lean documentation" works, but not the infoview.

Do you have any suggestions for how to debug this? (I am a programmer, but my expertise is in C++. I'm not familiar with React.)

Here is a screenshot:
<img width="955" alt="image" src="https://user-images.githubusercontent.com/15963396/221395116-1620b485-34f5-4f66-b57a-b2bd54dc4e7f.png">

**Comment by Vtec234** [comment]
Hi! One thing you could try would be to run `Developer: Toggle Developer Tools` from the command panel and paste the contents of the `Console` tab here (using `<details>` to avoid a huge message). Then we can look for anything suspicious there.

**Comment by PDeets** [comment]
I did not get it to work on my Windows ARM64 device. (It is a [Windows Dev Kit 2023](https://www.microsoft.com/en-us/d/windows-dev-kit-2023/94K0P67W7581?activetab=pivot:overviewtab).) What I eventually did is buy a new computer with an Intel processor. It worked fine on the computer with the Intel processor.

**Comment by PDeets** [comment]
Yes, that is correct. The new computer where it worked is an HP Omen laptop running Windows 11.
